src/webhooks/spec/data/po.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
maybe it's ought to rename this file? because `po.json` means nothing to me. But from other point of view it's just one more stub data for test